src/ansible_navigator/actions/exec.py

```diff
@@ -0,0 +1,86 @@
+""" :exec """
+import os
+import shlex
+
+from typing import Optional
+from typing import Tuple
+from typing import Union
+
+from . import _actions as actions
+from ..app import App
+from ..runner import Command
+
+
+@actions.register
+class Action(App):
+    """:exec"""
+
+    # pylint: disable=too-few-public-methods
+
+    KEGEX = r"^e(?:xec)?$"
+
+    def __init__(self, args):
+        super().__init__(args=args, logger_name=__name__, name="exec")
+
+    @staticmethod
+    def _generate_command(exec_command: str, exec_shell: bool) -> Tuple:
+        """Generate the command and args"""
+        pass_through_args = None
+        if exec_shell and exec_command:
+            command = "/bin/bash"
```

Comment:
I think I might have had it `sh` at a point, but then decided it should be the same as the default command.(`bash`), so the behavior would be the same at the shell as it would with a command passed through.